src/targets/browser/index.jsx

```diff
@@ -21,6 +21,7 @@ import { StepperDialogProvider } from 'src/components/Contexts/StepperDialogProv
 import { ScannerI18nProvider } from 'src/components/Contexts/ScannerI18nProvider'
 import { ModalProvider } from 'src/components/Contexts/ModalProvider'
 import { PlaceholderModalProvider } from 'src/components/Contexts/PlaceholderModalProvider'
+import * as serviceWorkerRegistration from 'src/targets/browser/serviceWorkerRegistration'
```

Comment:
In previous PdC we learned that `import *` should be avoided for Threeshaking.

```suggestion
import { register as registerServiceWorker } from 'src/targets/browser/serviceWorkerRegistration'
```